query-connector/src/app/ui/designSystem/TabGroup/tabGroup.tsx

```diff
@@ -1,12 +1,15 @@
 "use client";
-import React from "react";
+import React, { useState } from "react";
 import Link from "next/link";
-import { usePathname } from "next/navigation";
 import classNames from "classnames";
+import styles from "./tabGroup.module.scss";
 
 export interface Tab {
+  access?: string[];
   label: string;
-  path: string;
+  path?: string;
```

Comment:
kept path here for future flexibility, but the content is now controlled by the onClick/renderContent methods